### DISCUSSION TITLE
No `Content-Disposition` header

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#95] [Issue]
[The documentation](https://keygen.sh/docs/api/cryptography/) says:

> When downloading a license file, the `Content-Disposition` header will contain our recommended filename.

However, when checking out a license under `v1/accounts/{account_id}/licenses/{license_id}/actions/check-out`, I don't see such a header. Is this info outdated?

**Comment by ezekg** [comment]
The `content-disposition` header is only included when using the `GET` method (download), which responds with the raw license file certificate, rather than a JSON license file object. The header doesn't make sense for JSON responses.

**Reply by nschloe** [reply]
Thanks, this fixes it!
